packages/services/src/kernel/default.ts

```diff
@@ -1164,6 +1178,22 @@ namespace DefaultKernel {
   function shutdownAll(settings?: ServerConnection.ISettings): Promise<void> {
     return Private.shutdownAll(settings);
   }
+
+  /**
+   * Arguments interface for the anyMessage signal.
+   */
+  export
+  interface IAnyMessageArgs {
```

Comment:
Why put this interface in `default.ts` rather than `kernel.ts` with the other message types?

Reply:
Thanks, I got confused about what depended on what.